common/rpc/test/rpc_localstore_tls_test.go

```diff
@@ -391,3 +471,7 @@ func (s *localStoreRPCSuite) TestServerTLSInternodeToFrontendAlt() {
 func (s *localStoreRPCSuite) TestMutualTLSSystemWorker() {
```

Comment:
existing tests already cover fallback, right? if so cool. if not, can we add a test for when no per-host cert exists